styles/CSD120Mobile.css

```diff
@@ -58,6 +58,8 @@
         margin-top:1em;
         cursor:pointer;
         border-top: solid cyan 5px;
+
+        transition: all 200ms ease-in-out;

```

Comment:
Set the transition on all three of `.burger`, `.burger::before`, and `.burger::after`